organization/Neuromatch_Tutorial_Format.ipynb

```diff
@@ -69,19 +69,21 @@
         "---\n",
         "#Tutorial Objectives\n",
         "\n",
-        "Use a line (---) separator from title block to objectives. You should briefly introduce your content here in a few sentences. In this tutorial, you will learn what an NMA tutorial notebook should look like. **You should make sure the notebook runs from start to finish (do restart and run all and make sure there are no errors)**\n",
+        "Use a line (---) separator from title block to objectives. You should briefly introduce your content here in a few sentences. In this tutorial, you will learn what an NMA tutorial notebook should look like. \n",
         "\n",
-        "Then have a few specific objectives for this tutorial:\n",
+        "Then have a few specific objectives for this tutorial, ideally [SMART goals](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5944406/#:~:text=1%20The%20mnemonic%20SMART%E2%80%94Specific,a%20well%2Dwritten%20learning%20objective.&text=Verbs%20such%20as%20understand%2C%20know,measurable%20and%20should%20be%20avoided.). By the end of this tutorial you will be able to:\n",
         "*   Understand the format of an NMA tutorial\n",
-        "*   Learn about code exercises, interactive demos, and group discussions\n",
+        "*   Make code exercises, interactive demos, and group discussions the correct structure\n",
+        "\n",
+        "**For production editors: make sure to clear outputs before pushing to git**\n",
```

Comment:
CI handles this?

Reply:
the clearing outputs part? ah I didn't realize that

Reply:
https://github.com/NeuromatchAcademy/course-content/blob/master/ci/process_notebooks.py#L254-L299

Reply:
(Though if DL is not going to use the CI, they'll need to remember to do this)